Moller-Plesset/MP2_Hessian.py

```diff
@@ -0,0 +1,776 @@
+# -*- coding: utf-8 -*-
+"""
+This script calculates nuclear hessians for MP2 using the
+gradients of one and two electron integrals obtained from PSI4. 
+
+References: 
+1. "Derivative studies in hartree-fock and møller-plesset theories",
+J. A. Pople, R. Krishnan, H. B. Schlegel and J. S. Binkley
+DOI: 10.1002/qua.560160825
+
+2. "Analytic evaluation of second derivatives using second-order many-body 
+perturbation theory and unrestricted Hartree-Fock reference functions",
+J. F. Stanton, J. Gauss, and R. J. Bartlett
+DOI: 10.1016/0009-2614(92)86135-5
+
+3. "Coupled-cluster open shell analytic gradients: Implementation of the
+direct product decomposition approach in energy gradient calculations",
+J. Gauss, J. F. Stanton, R. J. Bartlett
+DOI: 10.1063/1.460915
+"""
+
+__authors__ = "Kirk C. Pearce"
+__credits__ = ["Kirk C. Pearce", "Ashutosh Kumar"]
+__copyright__ = "(c) 2014-2017, The Psi4NumPy Developers"
+__license__ = "BSD-3-Clause"
+__date__ = "2019-08-15"
+
+import time
+import numpy as np
+import copy
+import psi4
+from psi4 import *
+
+# Setup NumPy options
+np.set_printoptions(
+    precision=15, 
+    linewidth=200, 
+    suppress=True, 
+    threshold=np.nan
```

Comment:
this is causing your error. replace `np.nan` with `np.maxsize`

Reply:
Thanks. Took me a bit, but I just figured that out that's what was causing it almost simultaneously.

Reply:
Hmm “AttributeError: module 'numpy' has no attribute 'maxsize'”

Reply:
whoops, `sys.maxsize`

Reply:
That fixed my 37 fail, but now I'm failing 36 for some reason. Is there a way to get Travis to show more useful output about why you're failing?